### PR TITLE
Setup and mobile version skeleton

### PR DESCRIPTION
**Pull request by ooshoke**
- Add Html and CSS for Toolbar
-  Add Html and CSS for Headline
- Install linters

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins&display=swap" rel="stylesheet">
+</head>
+<body>
+    <section class="flexcontainer">
+        <nav class="toolbar">
+            <img src="img/image_geometry_1.svg" alt="">
+            <img src="img/ic_menu.svg" alt="">
+        </nav>
```
Comment (danmainah) [inline index.html]: - [x] Kindly make sure this section has a header tag container, currently, you have a nav tag container.
- [x] Kindly make sure only the hamburger button is on the right and has a nav tag, also check on the hamburger menu positioning there needs to be some margin on the right to match the design given.
image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+        <section class="flexcontainer">
+            <h1 class="topic">Okolo <br> Oshoke</h1>
+            <p class="text1">
+                Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.
+                If you like what you see and have a project you need coded, don’t hesitate and contact me.
+            </p>
+            <button class="button1">Start collaboration</button>
```
Comment (danmainah) [inline index.html:26]: Kindly make sure your hover effect is similar to the design given, below is the design given;
image

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @ooshoke, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design
✔️  Good PR and Description


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 1 above)
Comment (ooshoke) [reply index.html]: Will do

File: index.html (hunk 2 above)
Comment (ooshoke) [reply index.html:26]: About this, I wasn't too clear cause we were asked to build the mobile version and I didn't get any changes when I added the button interaction for hover but I added the interaction for pressed or that doesn't matter?

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Lorenzo, 
## Congratulations 🎉 Thank you for implementing all the requirements 🙏
### Please go ahead and merge the branch!

## Happy Coding 👏👏👏
#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**